ui/perfherder/tests/AlertsLink.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import SimpleTooltip from '../../shared/SimpleTooltip';
+
+export default class AlertsLink extends React.PureComponent {
+  render() {
+    const { alerts, framework: frameworkName, frameworks } = this.props;
+    const {
+      total_alerts: totalAlerts,
+      total_regressions: regressions,
+      suite,
+      test,
+    } = alerts;
+    const improvements = totalAlerts - regressions;
+    const framework = frameworks.find((item) => item.name === frameworkName);
+
+    const filterText = suite && test ? suite.concat(`+${test}`) : suite || test;
+    return (
+      <div className="d-flex justify-content-center">
+        <div data-testid="improvements" className="w-50">
+          <Link
+            to={`./alerts?hideDwnToInv=0&filterText=${filterText}&page=1&status=4&framework=${framework.id}`}
```

Comment:
Instead of using status=4 or status=9 in the url created, maybe we can use the status object and make it more clear on what that status number actually means? So instead of status=4 it can look like status=${status.untriaged}